modules/op-tkn-enabling-tab-completion.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="op-tkn-enabling-tab-completion_{context}"]
+= Enabling tab completion
+
+After you install the `tkn` CLI, you can enable tab completion to automatically complete `tkn` commands or suggest options when you press Tab.
+
+.Prerequisites
+
+* You must have the `tkn` CLI tool installed.
+* You must have `bash-completion` installed on your local system.
+
+.Procedure
+
+The following procedure enables tab completion for Bash.
+
+. Save the Bash completion code to a file.
++
+----
+$ tkn completion bash > tkn_bash_completion
+----
+
+. Copy the file to `/etc/bash_completion.d/`.
++
+----
+$ sudo cp tkn_bash_completion /etc/bash_completion.d/
+----
++
+You can also save the file to a local directory and source it from your `.bashrc` file instead.
```

Comment:
I would suggest rewriting for clarity, maybe along the lines of:
"Alternately, you can save the file to a local directory and source it from your `.bashrc` file instead."

Reply:
done